### ISSUE TITLE
Json master data structure tests

### ISSUE THREAD

**Issue by banillie** [banillie/analysis_engine#20]
Look at and potentially refactor the process for storing master data in a json data structure format.

**Comment by banillie** [comment]
#17

**Comment by yulqen** [comment]
`bcompiler-engine` already temporarily creates a JSON structure before building the master spreadsheet. It shouldn't be a massive effort to create a new output format for `datamaps`, to produce a JSON file, which you can do what you like with in `analysis_engine`.  We need to discuss this week if possible.

**Comment by banillie** [comment]
Great. You've preempted me asking you how how bcompiler-engine handles json data files. I tried putting a Datamaps Master into a json file and got an error message because it's a Class. But, I also got an error when I tried to dump the Master.data dict into json, so probably doing something wrong there. 
There is a lot of useful information in the Master Class though e.g projects and quarter, which it would be good to have access to you. 
Anyway yes let's discuss. This isn't something I was going to do immediately, more of a medium term improvement and I won't start on it until we've discussed.
